components/feedback/child_components/form/wteg_feedback_form/wteg_feedback_form.react.js

```diff
@@ -0,0 +1,166 @@
+import React, {Component} from 'react';
+import Button from '../../button/details_feedback_widget_button.react';
+import Fieldset from '../../fieldset/details_feedback_fieldset.react';
+import Comment from "../../comment/details_feedback_comment.react";
+import { Data } from '../../../data';
+import Message from "../../message/details_feedback_message.react";
+
+class WTEGFeedbackForm extends Component{
+	constructor(props){
+		super(props);
+
+		this.buildGtmObj = this.buildGtmObj.bind(this);
```

Comment:
Could we use arrow functions for the methods? Then we will not need to bind them, as arrow functions will have its own scope.

Reply:
Done